F1/F1/Standings/StandingsViewController.swift

```diff
@@ -7,17 +7,18 @@
 
 import UIKit
 
-class StandingsViewController: UIViewController {
+class StandingsViewController: LoadingIndicatorViewController {
 
     // MARK: IBOutlets
     @IBOutlet weak private var tableView: UITableView!
-    @IBOutlet weak private var segControl: UISegmentedControl!
+    @IBOutlet weak private var segmentedControl: UISegmentedControl!
 
     // MARK: Variables
     var selectedSegmentIndex = 1
     private lazy var driverViewModel = DriverStandingsViewModel(repository: DriverStandingsRepository(), delegate: self)
     private lazy var constructorViewModel = ConstructorStandingsViewModel(repository: ConstructorStandingsRepository(), delegate: self)
     private lazy var standingsViewModel = StandingsViewModel(driverViewModel: driverViewModel, constructorViewModel: constructorViewModel, navigationDelegate: self)
+    private let dispatchGroup = DispatchGroup()
```

Comment:
What is this used for?

Reply:
Good catch, I was trying to use concurrency with the loading indicators to know when they should be dismissed. But decided to go a different route. Will remove it 😄